internal/sql/sql.go

```diff
@@ -58,10 +64,15 @@ func Connect(
 // the sqlcmd.Sqlcmd object. It sets the standard output and standard error
 // to be the same as the current process, and returns the error if any occurred
 // during the execution of the query.
-func Query(s *sqlcmd.Sqlcmd, text string) {
-	s.Query = text
-	s.SetOutput(os.Stdout)
-	s.SetError(os.Stderr)
-	err := s.Run(true, false)
-	checkErr(err)
+func (m *SqlType) Query(text string) {
+	if m.console == nil {
+		m.sqlcmd.Query = text
+		m.sqlcmd.SetOutput(os.Stdout)
+		m.sqlcmd.SetError(os.Stderr)
```

Comment:
See related comment about the `Output` package. Should this command route to the active `Output` instead of standard ?

Reply:
Right now because we haven't wired up all the Kong switches, there is no way to set different output/error targets, so I set it to these defaults now,  But when we start wiring up all the back-compat switches to the query subcommand these will have to change.